crates/host/src/wasmbus/mod.rs

```diff
@@ -2506,131 +2506,144 @@ impl Host {
 
         debug!(actor_ref, count, "scale actor");
 
-        let actor = self.fetch_actor(&actor_ref).await?;
-        let claims = actor.claims().context("claims missing")?;
-        let actor_id = claims.subject.clone();
-        let resp = self
-            .policy_manager
-            .evaluate_action(
-                None,
-                PolicyRequestTarget::from(claims.clone()),
-                PolicyAction::StartActor,
-            )
-            .await?;
-        if !resp.permitted {
-            bail!(
-                "Policy denied request to start actor `{}`: `{:?}`",
-                resp.request_id,
-                resp.message
-            )
-        };
+        let host_id = host_id.to_string();
+        spawn(async move {
```

Comment:
Can we introduce a `handle_scale_actor_task` like for `launch_actor`?

Reply:
Sure 👍

Reply:
Done!